src/actions/configuration/first-time-configuration-action.php

```diff
@@ -81,6 +91,15 @@ public function set_site_representation( $params ) {
 		$this->options_helper->set( 'company_logo_meta', false );
 		$this->options_helper->set( 'person_logo_meta', false );
 
+		/**
+		 * Action: 'wpseo_post_update_site_representation' - Allows for Hiive event tracking.
+		 *
+		 * @param array The new values of the options.
+		 * @param array The old values of the options.
+		 * @param array The options that failed to be saved.
+		 */
+		\do_action( 'wpseo_post_update_site_representation', $params, $old_values, $failures );
```

Comment:
* You used in the add_action `wpseo_ftc_post_update_site_representation` elsewhere!
* Maybe signal this as `@internal`? Or create documentation PR?

Reply:
Done.